src/scripts/windows/prepare-env.sh

```diff
@@ -93,6 +93,9 @@ if ! resolve_unity_serial; then
   exit 1
 fi
 
+# Create folder to store the build artifacts.
+mkdir -p "$base_dir/build"
```

Comment:
We might want to toss on something like: 
```suggestion
mkdir -p "$base_dir/build" || { echo "Unable to create the build directory"; exit 1; }
```